<launch>
	<node name="interface_RS485"
	    pkg="interface_rs485"
	    type="interface_rs485_node"
	    output="screen" />
</launch>
